vm/isolated/isolated.go

```diff
@@ -331,8 +363,40 @@ func (inst *instance) Run(timeout time.Duration, stop <-chan bool, command strin
 	return vmimpl.Multiplex(cmd, merger, dmesg, timeout, stop, inst.closed, inst.debug)
 }
 
+func (inst *instance) readPstoreContents() ([]byte, error) {
+	log.Logf(0, "reading pstore contents")
+	args := append(vmimpl.SSHArgs(inst.debug, inst.sshKey, inst.targetPort),
+		inst.sshUser+"@"+inst.targetAddr, "cat "+pstoreConsoleFile)
```

Comment:
We can add + "&& rm " + pstoreConsoleFile at the end of the command and then we don't need the second ssh connection.